### PR TITLE
Nextcloud takeover

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add new Nextcloud takeover

## QA

Check the takeover in s/m and l

## Card and doc

Card: https://trello.com/c/Mo9nrFpZ
Doc: https://docs.google.com/document/d/1xMTbFdLr7dhlrfxgdnZjxLIFHuNMpjwWON6Ma8Loa_4/edit?ts=57d675e9

### CODE REVIEW

File: static/css/section/_homepage.scss [hunk 1]
```diff
@@ -1614,3 +1614,64 @@ body.homepage-linux-25th-takeover {
...
+      background-size: cover;
+      background-position: center bottom;
+      color: $white;
+      margin-top: 0;
+
+      .row-hero__heading {
+        margin-top: 20px;
+      }
+
+      .row-hero___image {
```
Comment (anthonydillon) [inline static/css/section/_homepage.scss]: Same here.

File: static/css/section/_homepage.scss [hunk 2]
```diff
@@ -1614,3 +1614,64 @@ body.homepage-linux-25th-takeover {
...
+        margin-top: 10px;
+        margin-bottom: 20px;
+      }
+      
+      @media only screen and (min-width : 768px) {
```
Comment (anthonydillon) [inline static/css/section/_homepage.scss]: Can you move this query above the child classes above.

File: static/css/section/_homepage.scss [hunk 3]
```diff
@@ -1614,3 +1614,64 @@ body.homepage-linux-25th-takeover {
...
+        background-image: url('#{$asset-server}5aa79282-56f66029-Nextbox+09.2016.desktop.background-large.png');
+        min-height: 440px;
+
+        .row-hero__heading {
```
Comment (anthonydillon) [inline static/css/section/_homepage.scss]: Can you put these as separate media queries inside the child classes please.

File: static/css/section/_homepage.scss [hunk 4]
```diff
@@ -1614,3 +1614,64 @@ body.homepage-linux-25th-takeover {
...
+          margin-top: 60px;
+        }
+        
+        .row-hero___image {
```
Comment (anthonydillon) [inline static/css/section/_homepage.scss]: Please media query inside the single class above.

File: static/css/section/_homepage.scss [hunk 5]
```diff
@@ -1614,3 +1614,64 @@ body.homepage-linux-25th-takeover {
...
+        min-height: 540px;
+
+        .row-hero__heading {
+          margin-top: 60px;
+        }
+
+        .row-hero___image {
+          margin-top: 60px;
+        }
+      }
```
Comment (anthonydillon) [inline static/css/section/_homepage.scss]: Same for the code above.

File: static/css/section/_homepage.scss [hunk 6]
```diff
@@ -1614,3 +1614,64 @@ body.homepage-linux-25th-takeover {
...
+      
+      .button--primary {
+        background-color: $white;
+        color: $cool-grey;
+    
+        &:hover {
+          background-color: #f7f7f7;
```
Comment (anthonydillon) [inline static/css/section/_homepage.scss]: This should be `$light-grey`?

**Comment by deadlight** [comment]
Good code

**Pull request merged**